src/display/text_layer.js

```diff
@@ -189,7 +178,7 @@ const renderTextLayer = (function renderTextLayerClosure() {
     // little effect on text highlighting. This makes scrolling on docs with
     // lots of such divs a lot faster.
     let shouldScaleText = false;
-    if (geom.str.length > 1) {
+    if (geom.str.length > 1 || geom.isSpace) {
```

Comment:
Is it necessary to do this for *all* spaces, or was it only necessary for the `enhanced`-mode?
If the latter is true, then the following would probably be a good idea: 
```javascript
if (geom.str.length > 1 || (task._enhanceTextSelection && geom.isSpace)) {
```